Editor documentation for Engine V2 (#691)
playcanvas/developer.playcanvas.com: docs/user-manual/editor/editor-migrations.md

---
title: Editor Migrations
sidebar_position: 13
---

## Introduction

> This page is to outline the migrations of data between different editor versions

Reviewer: ```suggestion
This page outlines the migrations of data between different editor versions.
```

### Migration from 1.48.0 to 1.50.0

This Editor has been updated to now use Engine V2 internally. This change may cause some small visual changes for some projects. We have collated a list of the most common issues and how to fix them.

#### Cubemap edge filtering

<img src='/img/user-manual/editor/editor-v2/edge-filter.png' width='600' />

If your cubemap skybox has pronounced edges such as this example above, navigate to your cubemap asset and delete and regenerate the prefiltered data to remove them.

<img src='/img/user-manual/editor/editor-v2/prefiltered-data.png' width='400' />

#### Gamma Correction

<img src='/img/user-manual/editor/editor-v2/gamma-compare.png' />

> If you have a project with a gamma correction set to 1.0 your scene may appear more saturated such as the example above (New editor on the right). Under the rendering settings change your gamma correction to 2.2 to achieve a similar effect as before.

Reviewer: ```suggestion
If you have a project with gamma correction set to 1.0, your scene may appear more saturated such as the example above (new editor on the right). Under the rendering settings, change your gamma correction to 2.2 to achieve a similar effect as before.
```

> > **Note:** Your scene will be rendered with more correct linear workflow however there will be slight visual changes related to lighting and alpha blending.

Reviewer: ```suggestion
:::note

Your scene will be rendered with the more correct linear workflow. However, there will be slight visual changes related to lighting and alpha blending.

:::
```
